src/node_crypto.h

```diff
@@ -98,6 +98,7 @@ extern int VerifyCallback(int preverify_ok, X509_STORE_CTX* ctx);
 extern void UseExtraCaCerts(const std::string& file);
 
 void InitCryptoOnce();
+const std::string GetOpenSSLVersion();
```

Comment:
Btw, `const` on return types is ignored and at least some versions of GCC provide compiler warnings for that… might be best to just omit it.

Reply:
thanks, removed it.